R/DownloadModule.R

```diff
@@ -141,10 +141,13 @@ download_report_button_srv <- function(id,
         },
         content = function(file) {
           shiny::showNotification("Rendering and Downloading the document.")
+          shinybusy::block(id = ns("download_data"), text = "", type = "dots")
+          Sys.sleep(3)
```

Comment:
While it might help with the review - don't forget to revert it before merging!
Pls keep it unresolved as a reminder

Reply:
Of course :slightly_smiling_face: